### PR TITLE
Add PHP Core Roundup #8

### PR DESCRIPTION
**Pull request by Ayesh**

### CODE REVIEW

File: source/_posts/2022-11-30-php-core-roundup-8.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+* [Readonly class amendments](https://externals.io/message/119007)
+* [Asymmetric Visibility, with readonly](https://externals.io/message/118994)
+
+
+## Merged PRs and Commits
+
+Following are some changes that did not go through an RFC process because they are either planned, bug fixes, or progressive enhancements. There are automated unit and integration tests for each of these changes,  the PHP core developers review all pull requests.
+
+ - Fix stub type info for `posix_getrlimit` in [c3b9b0f9a7](https://github.com/php/php-src/commit/c3b9b0f9a7) by Ilija Tovilo 💜
+ - Fix pre-PHP 8.2 compatibility for `php_mt_rand_range()` with MT_RAND_PHP in [GH-9839](https://github.com/php/php-src/pull/9839) by Tim Düsterhus
```
Comment (saundefined) [inline source/_posts/2022-11-30-php-core-roundup-8.md:77]:
```suggestion
 - Fix pre-PHP 8.2 compatibility for `php_mt_rand_range()` with `MT_RAND_PHP` in [GH-9839](https://github.com/php/php-src/pull/9839) by Tim Düsterhus
```

File: source/_posts/2022-11-30-php-core-roundup-8.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+ - Fix duplicate SKIPIF section in [d2c663441d](https://github.com/php/php-src/commit/d2c663441d) by Ilija Tovilo 💜
+ - Fix ext section in [bca1e1f557](https://github.com/php/php-src/commit/bca1e1f557) by Ilija Tovilo 💜
+ - Disable CLI server `pdeathsig` test on 32-bit GitHub actions in [1fb40b501d](https://github.com/php/php-src/commit/1fb40b501d) by Ilija Tovilo 💜
+ - Fix generator memory leaks when interrupted during argument evaluation in [GH-9756](https://github.com/php/php-src/pull/9756) by Arnaud Le Blanc 💜
+ - Fix memory leak in [f31f464cec](https://github.com/php/php-src/commit/f31f464cec) by Dmitry Stogov
+ - Don't check "fake" closures (fix assertion) in [05b63b1593](https://github.com/php/php-src/commit/05b63b1593) by Dmitry Stogov
+ - Fiber: add shadow stack support in [GH-9283](https://github.com/php/php-src/pull/9283) by Chen, Hu
+ - Don't skip test on Windows due to missing `ext/posix` in [GH-9886](https://github.com/php/php-src/pull/9886) by Christoph M. Becker
+ - Fix cross-compilation for copy_file_range in [5d7b64be1d](https://github.com/php/php-src/commit/5d7b64be1d) by Bob Weinand
```
Comment (saundefined) [inline source/_posts/2022-11-30-php-core-roundup-8.md:112]:
```suggestion
 - Fix cross-compilation for `copy_file_range` in [5d7b64be1d](https://github.com/php/php-src/commit/5d7b64be1d) by Bob Weinand
```

File: source/_posts/2022-11-30-php-core-roundup-8.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+{% include "newsletter.html" %}
+
+## The PHP Foundation: Impact and Transparency Report 2022
+
+[Roman Pronskiy](https://twitter.com/pronskiy) wrote a detailed post on PHP Foundation’s impact, timeline, a summary of contributions made to PHP by PHP Foundation team, governance, financials, and more on [The PHP Foundation: Impact and Transparency Report 2022](https://thephp.foundation/blog/2022/11/22/transparency-and-impact-report-2022/).
+
+
+## PHP 8.2 to be released on December 8
+
+PHP 8.2 was scheduled to be released on 24th of November. However, because it falls on the US thanksgiving holiday, and to take some time to fix some bugs reported recently, PHP 8.2’s release managers decided to postpone the PHP 8.2 GA release to 8th of December.
```
Comment (saundefined) [inline source/_posts/2022-11-30-php-core-roundup-8.md:29]: Maybe add ref to https://externals.io/message/118991 ?

File: source/_posts/2022-11-30-php-core-roundup-8.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+
+## PHP 8.2 to be released on December 8
+
+PHP 8.2 was scheduled to be released on 24th of November. However, because it falls on the US thanksgiving holiday, and to take some time to fix some bugs reported recently, PHP 8.2’s release managers decided to postpone the PHP 8.2 GA release to 8th of December. 
+
+
+## All PHP 7 versions are now End-of-Life
+
+PHP 7.4 is the last version in the PHP 7.4 series, and it reached its end of life on November 28th. This means that there will be no more bugs or security fixes. If you are running any PHP applications on PHP 7 make sure to upgrade to PHP 8, or to obtain Long-term support from a vendor. PHP 7 [still holds](https://php.watch/news/2021/11/php7-eol#stats) (based on [Packagist.org](https://packagist.org/php-statistics) and[ WordPress.org](https://wordpress.org/about/stats/) data) the major usage share, so this is alarming.
```
Comment (saundefined) [inline source/_posts/2022-11-30-php-core-roundup-8.md:34]:
```suggestion
PHP 7.4 is the last version in the PHP 7.4 series, and it reached its end of life on November 28th. This means that there will be no more bugs or security fixes. If you are running any PHP applications on PHP 7 make sure to upgrade to PHP 8, or to obtain Long-term support from a vendor. PHP 7 [still holds](https://php.watch/news/2021/11/php7-eol#stats) (based on [Packagist.org](https://packagist.org/php-statistics) and [WordPress.org](https://wordpress.org/about/stats/) data) the major usage share, so this is alarming. 
```

**Review by saundefined — COMMENTED** [review]
Thanks!

Minor nits, what do you think?

**Comment by Ayesh** [comment]
Awesome, thanks a lot @saundefined. I added all of your suggestions as fixup with you as a co-author, and force pushed.

**Pull request merged**